.travis.yml

```diff
@@ -12,12 +12,7 @@ install:
     - pip install yapf==0.6.2
     - pip install pyflakes==1.0.0
 script:
-    # Run unit tests and calculate code coverage.
-    - coverage run --source client_wrapper -m unittest discover
-    # Check that source has correct formatting.
-    - yapf --diff --recursive --style google ./
-    # Run static analysis for Python bugs/cruft.
-    - pyflakes client_wrapper/*.py tests/*.py
+    ./build
```

Comment:
Yeah, it looks weird to me too, but style guide discourages .sh

>Executables should have no extension (strongly preferred) or a .sh extension. Libraries must have a .sh extension and should not be executable.
It is not necessary to know what language a program is written in when executing it and shell doesn't require an extension so we prefer not to use one for executables.

https://google.github.io/styleguide/shell.xml?showone=File_Extensions#File_Extensions